routing_model/src/state.rs

```diff
@@ -73,6 +78,10 @@ impl State {
             return Some(next);
         }
 
+        if let Some(next) = self.as_process_elder_change().try_next(event) {
```

Comment:
This is not correct, should only run if active 
see
```
        if src.sub_routine_try_relocating.is_some() {
            if let Some(next) = self.as_try_relocating().try_next(event) {
                return Some(next);
            }
        }
```

Reply:
Good catch, this fixed most (except 2) test failures